test/api/oran_o2ims_suite_test.go

```diff
@@ -0,0 +1,324 @@
+package main_test
+
+import (
+	"bytes"
+	"os"
+	"testing"
+	"crypto/tls"
+	"io"
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestOranO2ims(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "OranO2ims Test Suite")
+}
+
+var _ = BeforeSuite(func() {
+	Expect(os.Getenv("TEST_HOST")).NotTo(BeZero(), "Please make sure TEST_HOST is set correctly.")
```

Comment:
At this point we don't have the infrastructure to run these tests in Prow or GitHub actions, so we will need to skip them when the environment variable isn't set, something like this:

```go
if os.Getenv("TEST_HOST") == "" {
        Skip("API tests were skipped because environment variable 'TEST_HOST' isn't set")
}
```

Can you do that change?